Modules/Registration/GPUPDEDeformable/include/itkGPUPDEDeformableRegistrationFilter.hxx

```diff
@@ -90,7 +90,6 @@ GPUPDEDeformableRegistrationFilter<TFixedImage, TMovingImage, TDisplacementField
   }
 
   os << indent << "SmoothingKernels: ";
-  if (m_SmoothingKernels != nullptr)
   {
```

Comment:
Sorry to chime in. These braces are then unnecessary.

Reply:
Yeah, but the point is I'm not even sure this change is correct at all. I'm not convinced these things can never be null.

Reply:
Top-level variables are statically allocated at the point of declaration:
https://github.com/InsightSoftwareConsortium/ITK/blob/v5.4.0/Modules/Registration/GPUPDEDeformable/include/itkGPUPDEDeformableRegistrationFilter.h#L222-L229

Reply:
But non-null check should go 4 lines down, here: `os << indent.GetNextIndent() << m_SmoothingKernels[d] << std::endl;`

Reply:
Oh, I think I see. So `m_SmoothingKernels` indeed cannot be null, but `m_SmoothingKernels[x]` might be, for some `x`?

Reply:
Exactly.

Reply:
Might be. There's no check currently, so it would be crashing already I suppose.

Reply:
Printing a null pointer results in something like `0x00000000`.